service/src/test/java/bio/terra/catalog/rawls/RawlsServiceTest.java

```diff
@@ -109,4 +116,60 @@ void userHasActionException() throws Exception {
         .thenThrow(new ApiException());
     assertThrows(RawlsException.class, () -> rawlsService.getRole(user, id));
   }
+
+  @Test
+  void entityQuery() throws Exception {
+    String id = "abc";
+    String name = "name";
+    String namespace = "namespace";
+    String tableName = "table";
+    WorkspaceResponse response =
+        new WorkspaceResponse().workspace(new WorkspaceDetails().name(name).namespace(namespace));
+    when(workspacesApi.getWorkspaceById(id, List.of())).thenReturn(response);
+    EntityQueryResponse queryResponse = new EntityQueryResponse();
+    when(entitiesApi.entityQuery(
+            namespace, name, tableName, null, null, null, null, null, List.of(), null, null))
+        .thenReturn(queryResponse);
+    assertThat(rawlsService.entityQuery(user, id, tableName), is(queryResponse));
+  }
+
+  @Test
+  void entityQueryException() throws Exception {
+    String id = "abc";
+    String name = "name";
+    String namespace = "namespace";
+    String tableName = "table";
+    WorkspaceResponse response =
+        new WorkspaceResponse().workspace(new WorkspaceDetails().name(name).namespace(namespace));
+    when(workspacesApi.getWorkspaceById(id, List.of())).thenThrow(new ApiException());
+    when(entitiesApi.entityQuery(
+            namespace, name, tableName, null, null, null, null, null, List.of(), null, null))
+        .thenThrow(new ApiException());
```

Comment:
I'm a bit surprised that this passes with strict mocks enabled, since I don't see how both mocks are called. Once `getWorkspaceById()` throws, `entitiesApi.entityQuery()` won't be called.